translational_speed: 15
goal_tolerance: 0.0001
time_at_waypoints: 0.0
waypoints: [
  [1.0, 0.0],
  [0.0, 0.0],
  [-1.0, 0.0]
]
